frida-gum/src/interceptor.rs

```diff
@@ -208,9 +225,11 @@ impl<'a> Interceptor<'a> {
     /// # Safety
     ///
     /// Should only be called from within a hook or replacement function.
+    //
+    // TODO: Should this function exist given the hook is passed an InvocationContext?
```

Comment:
Yes. This function is useful in the case where you replace the target function, in which case your hook has the same signature as the target function and thus can't receive the `InvocationContext`. You can use this function to retrieve the context though...

Reply:
Ack. Removed the TODO. I've done some digging into gum and understand better how this all works.